README.md

```diff
@@ -6,168 +6,208 @@ level: 100
-
-An on demand video was created for the Build 2018 event, featuring this scenario and this sample. See the video [Building Web App Solutions With Authentication](https://channel9.msdn.com/Events/Build/2018/THR5001), and the associated [PowerPoint deck](http://video.ch9.ms/sessions/c1f9c808-82bc-480a-a930-b340097f6cc1/SigninworkandschoolMSAusersWebApp.pptx)
+> This is the second tutorial, building on top of the previous one (in the master branch) showing how to add sign-in to a Web application. This ones adds the part which is about calling the Microsoft Graph in a controller on behalf of the user
 
 ## How to run this sample
 
 To run this sample:
 
-> Pre-requisites: Install .NET Core (for example for Windows) by following the instructions at [.NET and C# - Get Started in 10 Minutes](https://www.microsoft.com/net/core). In addition to developing on Windows, you can develop on [Linux](https://www.microsoft.com/net/core#linuxredhat), [Mac](https://www.microsoft.com/net/core#macos), or [Docker](https://www.microsoft.com/net/core#dockercmd).
+> Pre-requisites: go through how to run this sample from the master branch. This page only shows the increment.
 
 ### Step 1: Register the sample with your Azure AD tenant
 
-1. Sign in to the [Application registration portal](https://apps.dev.microsoft.com/portal/register-app) either using a personal Microsoft account (live.com or hotmail.com) or work or school account.
-1. Give a name to your Application, make sure that the *Guided Setup* option is **Unchecked**. Then press **Create**. The portal will assign your app a globally unique *Application ID* that you'll use later in your code.
-1. Click **Add Platform**, and select **Web**.
-1. In the Redirect URLs field, add `http://localhost:5000/` and `http://localhost:5000/signin-oidc`
+When you have registered your app as described in [the first tutorial](Step 1: Register the sample with your Azure AD tenant), you need an extra step:
 
-> Note: The base address in the **Sign-on URL** and **Logout URL** settings is `http://localhost:5000`. This localhost address allows the sample app to run insecurely from your local system. Port 5000 is the default port for the [Kestrel server](https://docs.microsoft.com/aspnet/core/fundamentals/servers/kestrel). Update these URLs if you configure the app for production use (for example, `https://www.contoso.com/signin-oidc` and `https://www.contoso.com/signout-oidc`).
+1. In the Application Secrets, section, click on **Generate New Password** and copy the value of the generated password. This value is now be needed for your Web App to call a Web API. Indeed, for this to happen, the Web App needs to get an access token for the Web API. This will be done using  MSAL.NET `ConfidentialClientApplication` and Confidential client application share a secret with Azure AD proving their identity.
 
-### Step 2: Download/ Clone this sample code or build the application using a template
+1. Save the changes in the page
```

Comment:
'1. Save the changes' only? I don't think we need In the page

Reply:
I agree. This change has been made.